config/pmix.m4

```diff
@@ -301,7 +301,8 @@ AC_DEFUN([PMIX_SETUP_CORE],[
                       time.h unistd.h dirent.h \
                       crt_externs.h signal.h \
                       ioLib.h sockLib.h hostLib.h limits.h \
-                      sys/statfs.h sys/statvfs.h])
+                      sys/statfs.h sys/statvfs.h \
+                      netdb.h ucred.h])
```

Comment:
I don't actually mind checking for netdb.h as the check is quick - just wondering why you added it since I don't see the result of that test being used?